.github/scripts/toolchain/group-gcc-test-failures.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+SCRIPT_DIRNAME=$(dirname ${BASH_SOURCE[0]})
+
+source $SCRIPT_DIRNAME/../config.sh
+
+TAG=$1
+GITHUB_STEP_SUMMARY=$2
```

Comment:
This script does not need to have notion of GitHub environment. For this reason, I'd rather name this file as `SUMMARY_FILE`.